common/rate-limiter/src/rate_limit.rs

```diff
@@ -239,19 +256,27 @@ impl Bucket {
             if self.allowed_in_period > 0 || self.throttled_in_period > 0 {
                 debug!(
                     throttle_label = self.label,
+                    throttle_log_info = self.log_info,
                     throttle_key = self.key,
                     num_allowed_in_period = self.allowed_in_period,
                     num_throttled_in_period = self.throttled_in_period,
+                    "{}-{}-{}={}/{}",
+                    self.label,
+                    self.log_info,
+                    self.key,
+                    self.allowed_in_period,
+                    self.allowed_in_period
+                        .saturating_add(self.throttled_in_period),
                 );
             }
 
             // Optional metrics
             if let Some(metrics) = self.metrics.as_ref() {
                 metrics
-                    .with_label_values(&[self.label.as_str(), self.key.as_str(), "allowed"])
+                    .with_label_values(&[self.label.as_str(), "allowed"])
```

Comment:
Any reason "key" was removed from the metrics?

Reply:
Too high cardinality :(

That would create a metric for every incoming IP ever connected to a node * the number of network interfaces.

I think in the future if we work some of those kinks out, this won't be an issue.

Reply:
:O Aah.. makes sense! Thanks @gregnazario 😄